static/src/javascripts-legacy/projects/common/modules/experiments/ab-test-clash.js

```diff
@@ -50,7 +50,10 @@ define([
 
     function userIsInAClashingAbTest(tests) {
         tests = tests || clashingTests;
-        return _testABClash(ab.isInVariant, tests);
+
+        var isUserInAClashingAbTest = _testABClash(ab.isInVariant, tests);
```

Comment:
Yeah you're right, this variable is a hangover from when userIsInAClashingAbTest resolved the check and was used in a conditional.